src/applications/letters/actions/letters.js

```diff
@@ -281,17 +288,16 @@ export function saveAddress(address) {
           response.data.attributes.address,
         );
         if (!isEqual(stripEmpties(address), stripEmpties(responseAddress))) {
-          const mismatchError = new Error(
-            "letters-address-update addresses don't match",
-          );
-          Raven.captureException(mismatchError);
+          throw new Error("letters-address-update addresses don't match");
```

Comment:
I'm pretty sure we're set up to have sentry catch all errors thrown by default, so no need to use `captureException` unless we're passing additional options to it